docker/driver.py

```diff
@@ -143,6 +145,12 @@ def terminate(job_name):
 print_help: bool = False
 if len(sys.argv) > 1:
     if sys.argv[1] == 'submit' or sys.argv[1] == 'start':
+        redis_exec = '/usr/bin/redis-server'
```

Comment:
Can we remove the hardcoded path? Instead, use some like os.environ["REDIS_PATH"]